docs/creating-a-runtime-check.md

```diff
@@ -0,0 +1,192 @@
+# Creating a Runtime Check
+
+Runtime checks are used to perform tests by executing the plugin's code.
+
+The Plugin Checker uses a number of interfaces to build out runtime checks.
+
+The Plugin Checker provides the `Runtime_Check` interface, which is used to identify a runtime check. This interface does not contain any methods but serves as a marker for runtime checks.
+
+```php
+use WordPress\Plugin_Check\Checker\Check_Result;
+
+class Custom_Check extends Runtime_Check {
+  public function run( Check_Result $result );
+    // Handle running the check and adding warnings or errors to the result.
+  }
```

Comment:
Needs to add `get_categories()` method.

Reply:
@mukeshpanchal27 I've updated the code examples with the `get_categories()` method where necessary